src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -875,8 +840,14 @@ def factoryWorkloadConstruction(self, workloadName, arguments):
 
         Named this way so that nobody else will try to use this name.
         """
-        self.masterValidation(schema=arguments)
-        self.validateSchema(schema=arguments)
+        if 'OriginalRequestName' not in arguments:
```

Comment:
One more comment that I noticed only now. We either have to
a) remove `OriginalRequestName` key/value from the request dictionary (only for clone, keep it for resubmission)
or
b) we have to add it to the StdBase create definition.

Otherwise it will fail the validation. If we go with b), then we may need to update this code and a few other places as well.